base/server/src/main/java/com/netscape/cms/servlet/request/CMSRequestDAO.java

```diff
@@ -63,6 +65,65 @@ public abstract class CMSRequestDAO {
     public CMSRequestDAO() {
     }
 
+    /**
+     * Finds list of requests matching the specified search filter.
+     *
+     * If the filter corresponds to a VLV search, then that search is executed and the pageSize
+     * and start parameters are used. Otherwise, the maxResults and maxTime parameters are
+     * used in the regularly indexed search.
+     *
+     * @param filter - ldap search filter
+     * @param start - start position
+     * @param pageSize - page size for search
+     * @param maxTime - max time for normal search
+     * @param uriInfo - uri context of request
+     * @return collection of key request info
+     * @throws EBaseException
+     */
+    public CMSRequestInfos listCMSRequests(String filter, RequestId start, int pageSize, int maxTime,
+            UriInfo uriInfo) throws EBaseException {
```

Comment:
Is this supposed to replace the old `listCMSRequests()` or add a new one? It looks like there is nothing using it right now.